sintia/sintia.py

```diff
@@ -342,13 +343,18 @@ def format_quote(self, quote: Quote) -> str:
         page_id, *rest = page_ids
         page_info = search_results['query']['pages'][page_id]
         paragraphs = page_info['extract'].split('\n')
-        return await message.channel.send(
-            f'**{page_info["title"]}**\n'
-            f'<{page_info["canonicalurl"]}>'
-            f'\n'
-            f'{paragraphs[0]}\n',
+        
+        embed = discord.Embed(
+            title=page_info['title'],
+            description=paragraphs[0],
+            url=page_info['canonicalurl'],
         )
 
+        if 'thumbnail' in page_info:
+            embed.set_thumbnail(url=page_info['thumbnail']['source'])
```

Comment:
Is 'source' guaranteed to be there? Same for other places. Unless you want to fail hard here.

Reply:
You're right, I could make this use `.get` and it'll return `None` to avoid issues in case there's a problem.